fuzz/Cargo.toml

```diff
@@ -11,13 +11,17 @@ edition = "2018"
 cargo-fuzz = true
 
 [dependencies]
-parsec = { path = "..", features = ["mbed-crypto-provider", "pkcs11-provider", "tpm-provider"] }
+parsec-service = { path = "..", features = ["mbed-crypto-provider", "pkcs11-provider", "tpm-provider"] }
 libfuzzer-sys = "0.3.0"
 flexi_logger = "0.14.5"
 log = "0.4.8"
 toml = "0.4.2"
 lazy_static = "1.4.0"
 arbitrary = { version = "0.4.0", features = ["derive"] }
+mockstream = "0.0.3"
```

Comment:
Actually I removed this dependency because we'd have to implement `Arbitrary` by hand on the mockstream if we use this one, so it doesn't make much of a difference.